packages/mui-base/src/TextareaAutosize/TextareaAutoSize.test.tsx

```diff
@@ -13,29 +13,38 @@ import TextareaAutosize from '@mui/base/TextareaAutosize';
 describe('<TextareaAutosize />', () => {
   const { clock, render } = createRenderer();
 
-  describeConformance(<TextareaAutosize />, () => ({
-    inheritComponent: 'textarea',
-    refInstanceof: window.HTMLTextAreaElement,
-    skip: [
-      'rootClass',
-      'componentProp',
-      'componentsProp',
-      'themeDefaultProps',
-      'themeStyleOverrides',
-      'themeVariants',
-    ],
-  }));
+  describeConformance(
+    <TextareaAutosize minRows={1} />,
+    () =>
+      ({
+        inheritComponent: 'textarea',
+        refInstanceof: window.HTMLTextAreaElement,
+
+        skip: [
+          'rootClass',
+          'componentProp',
+          'componentsProp',
+          'themeDefaultProps',
+          'themeStyleOverrides',
+          'themeVariants',
+        ],
+      } as any),
+  );
 
   describe('layout', () => {
-    const getComputedStyleStub = {};
+    const getComputedStyleStub: any = {};
     function setLayout(
-      input,
-      shadow,
-      { getComputedStyle, scrollHeight, lineHeight: lineHeightArg },
+      input: HTMLInputElement,
+      shadow: HTMLTextAreaElement,
+      {
+        getComputedStyle,
+        scrollHeight,
+        lineHeight: lineHeightArg,
+      }: { getComputedStyle: object; scrollHeight: number; lineHeight: number | (() => number) },
     ) {
       const lineHeight = typeof lineHeightArg === 'function' ? lineHeightArg : () => lineHeightArg;
 
-      getComputedStyleStub[input] = getComputedStyle;
+      getComputedStyleStub[input as any] = getComputedStyle;
```

Comment:
Something is not right here. An `HTMLInputElement` shouldn't be used as an index.